.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
 
 matrix:
   allow_failures:
-    - php: 7.0
+    - php: hhvm
```

Comment:
I'm more in the thinking of providing no support for hhvm, i really dont know anyone who is using it, and with upcoming php7 it may be even worse. 
IMO I don't want to pass too many hours supporting something not used at all (and rather use them to write more things for php-http :) ).